model-optimizer/extensions/front/tf/InterpolateTransposes.py

```diff
@@ -70,8 +70,18 @@ def replace_sub_graph(self, graph: Graph, match: dict):
         transpose_1 = match['transpose_1']
         transpose_2 = match['transpose_2']
 
+        axes = Interpolate.get_axes(interpolate)
```

Comment:
this transformation must be called after `InterpolateTranspose` so it works with `Interpolate-4` only.

Reply:
Why `InterpolateTransposes`  is called after `InterpolateTransposes`?

Reply:
According to the specification of Interpolate-4, Interpolate-4 operation has 'axes' as an optional **input** , but not as an attribute.